core/function.rs

```diff
@@ -281,7 +280,7 @@ pub enum Func {
     Math(MathFunc),
     #[cfg(feature = "json")]
     Json(JsonFunc),
-    Extension(ExtFunc),
+    Extension(crate::ext::ExternalFunc),
     External(Rc<ExternalFunc>),
```

Comment:
No, we actually want to get rid of `Extension` variant altogether and convert functions to use `External` instead. Note that this likely requires some more work on `ExternalFunc` so not a trivial task.